### PR TITLE
add ubuntu xenial packaging files

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

**- What I did**

Added a `debian/` directory which contains deb packaging files (currently meant for Ubuntu Xenial)

**- How I did it**

Magic :)

**- How to verify it**

    $ git archive --format=tar.gz --prefix=build-dir/ HEAD > ../cri-o_1.9.2.orig.tar.gz
    (tarball needs to be in same dir as source AFAIK)
    $ dpkg-buildpackage -us -uc

**- Description for the changelog**

Add packaging files for Ubuntu 16.04 LTS (Xenial)

Currently I have a `xenial` branch on my own fork which I rebase on the latest release tag whenever I need to do a new build.

If this commit gets in, I'm hoping that before cutting a new release, we can try a build of the deb package using the packaging files and bump `debian/changelog` as well, so my work gets reduced :) . As far as *trying* a build before release goes, I guess that could be done via a Makefile target that I had in https://github.com/kubernetes-incubator/cri-o/pull/924

@mrunalp @runcom @rhatdan PTAL

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Section: devel
+Priority: optional
+Maintainer: Lokesh Mandvekar <lsm5@fedoraproject.org>
+Build-Depends: debhelper (>=9),
+    rsync,
+    libdevmapper-dev,
+    libgpgme11-dev,
+    libseccomp-dev,
+    golang,
+    golang-1.8,
```
Comment (wking) [inline debian/control]: Do you need both `golang` and the versioned `golang-1.8`?  I'd expect `golang` would be sufficient.

File: debian/copyright [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cri-o
+Source: https://github.com/kubernetes-incubator/cri-o.git
+Comment:
+ All golang deps bundled.
+ FIles under vendor/ could have their own independent licenses.
```
Comment (wking) [inline debian/copyright]: “FIles” → “Files”.

If a deb package includes vendored Go dependencies, are you really allowed to ignore their licenses?

**Comment by runcom** [comment]
+1

**Comment by TomSweeneyRedHat** [comment]
Looks like some gofmt whitespace errors @lsm5 lsm5

**Comment by lsm5** [comment]
Closing this in favour of https://github.com/kubernetes-incubator/cri-o/pull/1380 . I fixed the copyright file there and addressed other comments (afaict).

**Pull request closed without merging**